src/Mvc/Mvc.Abstractions/src/ModelBinding/BindingInfo.cs

```diff
@@ -246,8 +247,16 @@ public bool TryApplyBindingInfo(ModelMetadata modelMetadata)
             PropertyFilterProvider = modelMetadata.PropertyFilterProvider;
         }
 
-        // There isn't a ModelMetadata feature to configure AllowEmptyInputInBodyModelBinding,
-        // so nothing to infer from it.
+        // If the EmptyBody behavior is not configured will be inferred
+        // as Allow when the NullablityState == NullablityStateNull or HasDefaultValue
+        // https://github.com/dotnet/aspnetcore/issues/39754
+        if (EmptyBodyBehavior == EmptyBodyBehavior.Default &&
+                BindingSource == BindingSource.Body &&
+                (modelMetadata.NullabilityState == NullabilityState.Nullable || modelMetadata.IsNullableValueType || modelMetadata.HasDefaultValue))
```

Comment:
Super nit: We usually let VS's formatting / indentation do it's thing. 


```suggestion
            BindingSource == BindingSource.Body &&
            (modelMetadata.NullabilityState == NullabilityState.Nullable || modelMetadata.IsNullableValueType || modelMetadata.HasDefaultValue))
```